BasicRdl/ViewModels/CategoryBrowserViewModel.cs

```diff
@@ -51,7 +51,7 @@ namespace BasicRdl.ViewModels
     /// <summary>
     /// Represents the view-model of the <see cref="CategoryBrowser"/> view
     /// </summary>
-    public class CategoryBrowserViewModel : BrowserViewModelBase<SiteDirectory>, IPanelViewModel, IDeprecatableBrowserViewModel
+    public class CategoryBrowserViewModel : BrowserViewModelBase<SiteDirectory>, IPanelViewModel, IDeprecatableBrowserViewModel, IPanelFilterableDataGridViewModel
```

Comment:
I wonder whether it would make sense to bring `IPanelFilterableDataGridViewModel` stuff up one level to `BrowserViewModelBase`. @lxatrhea @samatrhea? lots of filter code below is duplicated all over the place and in principle most if not all browsers should have filtering at some point.

Reply:
If all BrowserViewModelBase implementations can be considered filterable then it can be moved up to the base.